README.md

```diff
@@ -49,6 +49,25 @@ iex> ExHal.get_links_lazy(doc, "alternate", fn -> [] end)
 
 ```
 
+ExHal can also make requests. Continuing the example above:
+
+```elixir
+ExHal.follow_link(doc, "profile")
+{:error, %ExHal.Error{reason: "multiple choices"}}
+
+ExHal.follow_link(doc, "profile")
+{:error, %ExHal.Error{reason: "no such link"}}
+
+ExHal.follow_link("self")
+{:ok, %ExHal.Document{...}}
+
+ExHal.follow_link(doc, "profile", pick_volunteer: true)
```

Comment:
:smile: It means, if there are multiple links with the specify rel, pick one of them -- in an undefined way -- and follow it rather than blowing up (as shown in the "multiple choices" error example above). I'd like a better name if you can think of one.